.github/workflows/update-dependencies.yml

```diff
@@ -142,7 +142,7 @@ jobs:
               cd -
               # Copy all *-Swift.h header files into ios_pod/swift_headers/
               echo "Copying headers..."
-              find "${ziptmp}" -name '*-Swift.h' -print0 | xargs -0 -n 1 -J REPLACETEXT cp -f REPLACETEXT ios_pod/swift_headers/
+              find "${ziptmp}" -name '*-Swift.h' -path '*ios*arm64*x86_64*simulator*' -print0 | xargs -0 -n 1 -J REPLACETEXT cp -f REPLACETEXT ios_pod/swift_headers/
               copyright_line="// Copyright $(date +%Y) Google LLC"
```

Comment:
There is a github check warning that ios_pod/swift_headers/*.h no longer has the google copyright message (copyright_line), which the lines below this are supposed to add. Is it a side effect of this change, or is it something that is missing from the below command

Reply:
Ah yeah, this was omitted because I was manually testing different swift files rather than updating them via this command.

I added the missing copyright lines back in.